src/YQIconPool.cc

```diff
@@ -86,7 +86,7 @@ QPixmap
 YQIconPool::cachedIcon( const QString icon_name, const bool enabled )
 {
     Q_INIT_RESOURCE( icons );
-    QPixmap iconPixmap = _iconCache[ icon_name + enabled ];
+    QPixmap iconPixmap = _iconCache[ icon_name /*+ enabled*/ ];
```

Comment:
I am not even sure if that "enabled" thing is in use. It's very likely that we don't need it at all; that it was only needed in the broader context of the YaST Qt UI libyui-qt, not in libyui-qt-pkg, the package management part.

To do: Look at the `icons/` subdir if we even have enabled vs. disabled icons, and check where any disabled ones are used in the code, if they are used at all.